packages/next/src/server/lib/recursive-readdir-sync.ts

```diff
@@ -13,13 +13,12 @@ export function recursiveReadDirSync(
   /** Used to replace the initial path, only the relative path is left, it's faster than path.relative. */
   rootDir = dir
 ): string[] {
-  const result = fs.readdirSync(dir)
+  const result: fs.Dirent[] = fs.readdirSync(dir, { withFileTypes: true })
```

Comment:
```suggestion
  const result = fs.readdirSync(dir, { withFileTypes: true })
```

The type is inferred already so you can omit that

Reply:
Makes sense! Updated with a `fixup`